README.rst

```diff
@@ -43,15 +43,19 @@ Getting Started
 Include ``pinax-messages`` in your requirements file and add
 ``"pinax.messages"`` to your INSTALLED APPS setting.
 
-Once you have the ``pinax-messages`` installed, hook up the URLs::
+Run Django migration (``python manage.py migrate``) so that
+your project recognizes this new app.
```

Comment:
Thanks for fixing the urlpatterns construction @vleong2332.

Running migrations is not performed so the project "recognizes" the app, but so the pinax-messages database tables are created. Please reword this so inexperienced Django coders won't get confused. Thanks!